app.js

```diff
@@ -51,6 +51,14 @@ $(document).ready(() => {
         }
         $(".led").click(onLedClicked);
         $(".led").dblclick(() => $("#color").change());
+        $(".led").contextmenu(function (e) {
+            $(this).click();
+            e.preventDefault();
+            // Maybe is not the best
+            let last_color = $("#color").val();
+            $("#color").val('000000').change();
+            $("#color").val(last_color);
+        });
```

Comment:
Really like the changes. Perhaps reset frame should be a hotkey like CTRL+R or backspace? Right-click can be triggered accidentally and could be annoying especially with no undo functionality

Reply:
I can make another shortcut, here is a list of possible shortcuts that may be better than the right click:

 - Ctrl + R
 - Backspace
 - Long Click
 - Ctrl + Click
 - Space to reset the selected frame

Which is the best?

Reply:
I thought that you meant that when you pressed right-click *all* of the LEDs would go black. If it's just one led, I'd say right-click is fine.